src/forms/labelUtils.ts

```diff
@@ -0,0 +1,16 @@
+import { FieldState } from "@homebound/form-state";
+import { FormSettings } from "src/forms/FormContext";
+
+export function getLabelSuffix(settings: FormSettings, field: FieldState<any, any>): string {
+  switch (settings.labelSuffix) {
+    case "required":
+      return field.required ? "(Required)" : "";
+    case "optional":
+      return field.required ? "" : "(Optional)";
+    case "none":
+      return "";
+    case "both":
+      return field.required ? "(Req)" : "(Opt)";
```

Comment:
I know "Req" and "Opt" is probably too cute, but I've got a form where "Postal Code (Optional)" would wrap:

![image](https://user-images.githubusercontent.com/6401/126881617-d2e8aaed-d80c-4aea-a9fe-e41a6100570c.png)

Granted, I can one-off set the `labelSuffix` directly on this field, but, dunno, _maybe_ this is okay. :-) Especially in "both" mode seeing "Required" and "Optional" spelled out every gets kinda tedious anyway.

I'll also see what design thinks. I believe so far they've preferred spelling out "Required" and "Optional" instead of ambiguous (but short!) symbols like `*`, which length aside I generally agree with.